lib/ontologies_linked_data/models/ontology_submission.rb

```diff
@@ -81,12 +81,12 @@ class OntologySubmission < LinkedData::Models::Base
 
       # Person and organizations metadata
       attribute :contact, type: %i[contact list], enforce: [:existence]
-      attribute :hasCreator, namespace: :omv, type: %i[list Agent], enforce: [:is_person]
-      attribute :hasContributor, namespace: :omv, type: %i[list Agent], enforce: [:is_person]
+      attribute :hasCreator, namespace: :omv, type: %i[list Agent]
+      attribute :hasContributor, namespace: :omv, type: %i[list Agent]
       attribute :curatedBy, namespace: :pav, type: %i[list Agent]
       attribute :publisher, namespace: :dct, type: %i[list Agent]
-      attribute :fundedBy, namespace: :foaf, type: %i[list Agent], enforce: [:is_organization]
```

Comment:
here you should remove the implementation of the validators also, in a file called validation 
<img width="723" alt="image" src="https://github.com/user-attachments/assets/366748fb-400f-4782-9168-21a04131dd4b">

Reply:
and the `test_agent_no_valid` test should also not pass, the line 19

Reply:
`test_agent_no_valid` has no relation with this, we removed the validation for type of agents for the submission, and the test tests that for example the affiliation is of type organization

Reply:
is does have a relation, for example, this line, tests the `is_orgnanization` validator that does no more exists
![image](https://github.com/user-attachments/assets/ef25e791-8f30-4664-9dfa-2eb8549673bb)

Reply:
No it does exist, we removed it only for the submission,
This test is related to the object agent, and this pull request is related to ontology_submission object
![image](https://github.com/user-attachments/assets/5004d96b-215a-429f-9e67-c6ab01a60973)
![image](https://github.com/user-attachments/assets/6c2b0c48-bcb8-40da-ab3a-1f79f6bdfdfa)

Reply:
ah ok